.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I think we should also add mailing list for Gremlin users because very often there is a question about writing a query. And it should be asked in Gremlin google group. I.e. something like:
`General questions regarding the Gremlin graph traversal language in general should be raised in the [Gremlin users group](https://groups.google.com/forum/#!forum/gremlin-users).`

Reply:
Good idea